src/mono/mono/component/debugger-agent.c

```diff
@@ -5353,14 +5353,15 @@ decode_vtype (MonoType *t, MonoDomain *domain, gpointer void_addr, gpointer void
 	gpointer iter = NULL;
 	MonoDomain *d;
 	ErrorCode err;
+	int inlineArray = -1;
 
 	/* is_enum, ignored */
 	decode_byte (buf, &buf, limit);
 	if (CHECK_PROTOCOL_VERSION(2, 61))
 		decode_byte (buf, &buf, limit);
 	klass = decode_typeid (buf, &buf, limit, &d, &err);
 	if (CHECK_PROTOCOL_VERSION(2, 65))
-		decode_int (buf, &buf, limit); //ignore inline array
+		inlineArray = decode_int (buf, &buf, limit);
```

Comment:
What is this value? Is it the size/number of elements in the inline array? Naming it accordingly would be good.

Reply:
True, renaming to `inlineArraySize`